universal-application-tool-0.0.1/conf/messages.en-US

```diff
@@ -3,6 +3,7 @@
 button.logout=Logout
 button.apply=Apply
 button.nextBlock=Save and continue
+button.deleteEntity=Delete entity
```

Comment:
I don't see this one in the default `messages` file. Does it need to be added? Also, why are the deleteEntity and addEnumeratorEntity button messages separated - should they be grouped together?

Reply:
This is online 21 in `conf/messages`. 

There's gonna be some clean up to organize all the messages, so I'm just going to leave the messages where they are for now.